### ISSUE TITLE
Cannot find module 'strip-ansi'

### ISSUE THREAD

**Issue by bichikim** [wallabyjs/public#3343]
### Issue description or question
Wallabyjs Cannot find module 'strip-ansi' 
![2024-03-11 at 18 04](https://github.com/wallabyjs/public/assets/13829322/da75f7e1-82f9-42c7-b8d9-3c9f5b4aadf2)

### Wallaby diagnostics report

Copy Diagnostics Report is disabled

![2024-03-11 at 18 06](https://github.com/wallabyjs/public/assets/13829322/4be5b2a1-5060-4185-befc-1434a0928e15)

**Comment by evanliomain** [comment]
Same issue for me, just after the wallaby update :

```
‍​[Info]​ Started Wallaby.js Core v1.0.1535
​[Info]​ Wallaby App (realtime reports) is available at: http://localhost:51245
​​[Error] Failed to set up project file system: Cannot find module 'strip-ansi'​​
​​[Error] Require stack:​​
​​[Error] - [wallaby core]/server.js​​
​​[Error] Cannot find module 'strip-ansi'​​
​​[Error] Require stack:​​
​​[Error] - [wallaby core]/server.js​​
​​[Error]     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)​​
​​[Error]     at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (./node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)​​
​​[Error]     at Function.Module._load (node:internal/modules/cjs/loader:778:27)​​
​​[Error]     at Module.require (node:internal/modules/cjs/loader:1005:19)​​
​​[Error]     at Module.require (./node_modules/nx/src/adapter/compat.js:63:40)​​
​​[Error]     at require (node:internal/modules/cjs/helpers:102:18)​​
​​[Error]     at processTicksAndRejections (node:internal/process/task_queues:96:5)​​
```

In case it is relevant: my project stack :
- Angular: 17.1.2
- Nx: 18.0.1
- Jest: 29.7.0

**Comment by ArtemGovorov** [comment]
Thanks for reporting the issue. It's now fixed and the fix is published in the [the latest core v1.0.1536](https://wallabyjs.com/docs/intro/troubleshooting.html#forcing-wallabyjs-core-update).

**Comment by bichikim** [comment]
it's fixed

**Comment by evanliomain** [comment]
I also confirm it's fix in v1.0.1536.
Thank you for your reactivity 👍

**Comment by ArtemGovorov** [comment]
Awesome, thanks for the conformation! Apologies for the inconvenience caused by the regression.
